src/cli.js

```diff
@@ -9,398 +9,19 @@ import cleanStack from 'clean-stack';
-      );
-    } else if (commandMap.PROJECT_UPGRADE[subCommand]) {
-      return commands.projectUpgrade(
-        commands.toProjectUpgradeOptions(subCommandArgs, flags)
-      );
-    } else {
-      return commands.projectRun(
-        commands.toProjectRunOptions(commandArgs, flags)
-      );
-    }
-  } else if (commandMap.PUBLISH[command]) {
-    return commands.publish(commands.toPublishOptions(commandArgs, flags));
-  } else if (commandMap.PUBLISH_LOCK[command]) {
-    return commands.publishLock(
-      commands.toPublishLockOptions(commandArgs, flags)
-    );
-  } else if (commandMap.PUBLISH_UNLOCK[command]) {
-    return commands.publishUnlock(
-      commands.toPublishUnlockOptions(commandArgs, flags)
-    );
-  } else if (commandMap.REMOVE[command]) {
-    return commands.remove(commands.toRemoveOptions(commandArgs, flags));
-  } else if (commandMap.RUN[command]) {
-    return commands.run(commands.toRunOptions(commandArgs, flags));
-  } else if (commandMap.TAG[command]) {
-    if (commandMap.TAG_ADD[subCommand]) {
-      return commands.tagAdd(commands.toTagAddOptions(subCommandArgs, flags));
-    } else if (
-      commandMap.TAG_LIST[subCommand] ||
-      typeof subCommand === 'undefined'
-    ) {
-      return commands.tagList(commands.toTagListOptions(subCommandArgs, flags));
-    } else if (commandMap.TAG_REMOVE[subCommand]) {
-      return commands.tagRemove(
-        commands.toTagRemoveOptions(subCommandArgs, flags)
-      );
-    }
-  } else if (commandMap.TEAM[command]) {
-    if (commandMap.TEAM_ADD[subCommand]) {
-      return commands.teamAdd(commands.toTeamAddOptions(subCommandArgs, flags));
-    } else if (commandMap.TEAM_CREATE[subCommand]) {
-      return commands.teamCreate(
-        commands.toTeamCreateOptions(subCommandArgs, flags)
-      );
-    } else if (commandMap.TEAM_DESTROY[subCommand]) {
-      return commands.teamDestroy(
-        commands.toTeamDestroyOptions(subCommandArgs, flags)
-      );
-    } else if (
-      commandMap.TEAM_LIST[subCommand] ||
-      typeof subCommand === 'undefined'
-    ) {
-      return commands.teamList(
-        commands.toTeamListOptions(subCommandArgs, flags)
-      );
-    } else if (commandMap.TEAM_REMOVE[subCommand]) {
-      return commands.teamRemove(
-        commands.toTeamRemoveOptions(subCommandArgs, flags)
-      );
-    }
-  } else if (commandMap.TEST[command]) {
-    return commands.test(commands.toTestOptions(commandArgs, flags));
-  } else if (commandMap.UNLINK[command]) {
-    return commands.unlink(commands.toUnlinkOptions(commandArgs, flags));
-  } else if (commandMap.UPGRADE[command]) {
-    return commands.upgrade(commands.toUpgradeOptions(commandArgs, flags));
-  } else if (commandMap.UPGRADE_INTERACTIVE[command]) {
-    return commands.upgradeInteractive(
-      commands.toUpgradeInteractiveOptions(commandArgs, flags)
+  if (!command) return await commands['install'](flags, commandArgs);
+
+  if (command && commands[command])
```

Comment:
we will get command as:
```
{ 
  add: [add Function]
}
```

So we can just call is as commands['add']  -- given command is add ( bolt add ...  )

Reply:
we are passing all `flags, commandArgs, subCommand, subCommandArgs` but can ignore in commands?